installers/MacInstaller.hs

```diff
@@ -82,20 +83,30 @@ makeInstaller opts@Options{..} appRoot = do
   echo "Preparing files ..."
   case oAPI of
     Cardano -> do
+      -- fixme: when DEVOPS-690 adds option parsing, use a
```

Comment:
It did!

Reply:
OK, I have added it as a command line argument.